antwar.config.js

```diff
@@ -13,96 +13,152 @@ module.exports = {
   layout: () => require('./src/components/Site/Site.jsx').default,
   paths: {
     '/': {
-      content: () => (
-        require.context(
-          './loaders/page-loader!./src/content',
-          true,
-          /^\.\/.*\.md$/
-        )
-      ),
+      title: "Home",
+      content: () => require.context('./loaders/page-loader!./src/content', false, /^\.\/.*\.md$/),
       index: () => {
         const index = require('./src/components/Splash/Splash.jsx').default;
-
         index.title = 'webpack';
         index.description = 'webpack is a module bundler. Its main purpose is to bundle JavaScript files for usage in a browser, yet it is also capable of transforming, bundling, or packaging just about any resource or asset.';
 
         return index;
       },
       layout: () => require('./src/components/Page/Page.jsx').default,
-      paths: {
-        'get-started': {
-          redirects: {
-            '': '/guides/getting-started',
-            'install-webpack': '/guides/installation',
-            'why-webpack': '/guides/why-webpack',
-          }
-        },
-        guides: {
-          redirects: {
-            'code-splitting-import': '/guides/code-splitting-async',
-            'code-splitting-require': '/guides/code-splitting-async/#require-ensure-',
-            'why-webpack': '/guides/comparison',
-            'production-build': '/guides/production'
-          }
-        },
-        configuration: {
-          redirects: {
-            'external-configs': 'javascript-alternatives'
-          }
-        },
-        api: {
-          redirects: {
-            'passing-a-config': 'configuration-types'
-          }
-        },
-        pluginsapi: {
-          redirects: {
-            '': '/api/plugins',
-            'compiler': '/api/plugins/compiler',
-            'template': '/api/plugins/template'
-          }
-        },
-        loaders: {
-          content: () => {
-            const content = require.context(
-              './loaders/page-loader!./src/content/loaders',
-              false,
-              /^\.\/.*\.md$/
-            );
-            const generated = require.context(
-              './loaders/page-loader!./generated/loaders',
-              false,
-              /^\.\/.*\.md$/
-            );
-
-            return combineContexts(content, generated);
-          }
-        },
-        loaders: {
-          content: () => {
-            const content = require.context(
-              './loaders/page-loader!./src/content/plugins',
-              false,
-              /^\.\/.*\.md$/
-            );
-            const generated = require.context(
-              './loaders/page-loader!./generated/plugins',
-              false,
-              /^\.\/.*\.md$/
-            );
+    },
+    'get-started': {
+    	title: 'Get Gtarted',
+		redirects: {
+			'': '/guides/getting-started',
+			'install-webpack': '/guides/installation',
+			'why-webpack': '/guides/why-webpack',
+		}
+	},
```

Comment:
Formatting here and in the rest of the file is now a bit off which also makes it hard to see exactly what the changes were. It seems you were adding `title` and `transform` properties?

Reply:
Just add the title, I moved it out ` paths: {` of '/'.

Reply:
Ok, I think one of the initial points of this PR was to prevent manual configuration of each route which yields a lot of duplicate code, e.g. `transform` is pretty much the same in each place and detaches the routing from the actual content structure. I saw your other comment about the page sorting not working -- I'm going to wait until this CI build finishes and then play around with the `antwar.config.js` file a bit to see if we can clean things up.

Reply:
In terms of the formatting I mentioned -- please check ensure your editor is listening to the `.editorconfig` file. I just pulled your updates and this file in particular is mixed spaces and tabs now:

![image](https://user-images.githubusercontent.com/8988697/29897253-14fd4448-8dae-11e7-9dbc-c0c2cc564a4f.png)

Reply:
Totally agree 👍

Reply:
sorry for `.editorconfig`, I will check now.